src/worker.cc

```diff
@@ -40,6 +40,21 @@ Status WorkerServiceImpl::ImportFunction(ServerContext* context, const ImportFun
   return Status::OK;
 }
 
+Status WorkerServiceImpl::ImportReusableVariable(ServerContext* context, const ImportReusableVariableRequest* request, AckReply* reply) {
+  std::unique_ptr<WorkerMessage> message(new WorkerMessage());
+  message->reusable_variable = *(new ReusableVariable());
+  message->reusable_variable.variable_name = request->name();
+  message->reusable_variable.initializer = request->initializer().implementation();
+  message->reusable_variable.reinitializer = request->reinitializer().implementation();
+  RAY_LOG(RAY_INFO, "importing reusable variable");
+  {
+    WorkerMessage* message_ptr = message.get();
+    RAY_CHECK(send_queue_.send(&message_ptr), "error sending over IPC");
+  }
+  message.release();
+  return Status::OK;
+}
+
```

Comment:
Ok cool, looks like I can get rid of the line entirely (which I just did), thanks!